dmriprep/workflows/dwi/base.py

```diff
@@ -201,6 +203,43 @@ def _bold_reg_suffix(fallback):
         ])
         # fmt: on
 
+    # Eddy distortion correction
+    eddy_wf = init_eddy_wf(debug=config.execution.debug)
+    eddy_wf.inputs.inputnode.metadata = layout.get_metadata(str(dwi_file))
+
+    ds_report_eddy = pe.Node(
+        DerivativesDataSink(
+            base_directory=str(config.execution.output_dir),
+            desc="eddy_corrected",
+            datatype="figures",
+        ),
+        name="ds_report_eddy",
+        run_without_submitting=True,
+    )
+
+    eddy_report = pe.Node(
+        SimpleBeforeAfter(before_label="Distorted", after_label="Eddy Corrected",),
```

Comment:
I wonder whether there is a better reportlet. Although you can see that the average image seems to have a better SNR after eddy, do you think that some sort of standard deviation map across orientations or similar will give us a better picture of what this step did (and whether it did it well)?

WDTY @arokem ?

Reply:
The developers of eddy seem to like showing an animation of flipping through all the different directions rather quickly.

Reply:
Another option is to show before and after mean maps across all directions (is that what this does?). 

Unfortunately, I think that an std map would capture both artifactual and real variance (e.g., due to diffusion anisotropy).

Reply:
So, now that you made me check what this exactly does I can tell that it shows the b=0 reference we generate (averaging all b=0 in the dataset after regressing out signal drift) and the first volume of the eddy-corrected output (which happens to be a b=0, as it is the most common case). So at the very least, we want to plot two b=0 references generated the same way.

I'm not sure that we are going to see much averaging all directions... however, I believe the std map would be still useful - since you have the reference std map (that would be the "Before" on the flickering mosaic) and then you plot the std map after correction (the "After"), adjusted to have consistent intensity ranges, we should see how some of the variability (that from the artifact) goes away. Because of the structure of the artifact, I would expect better SNR (less noise in the "after" plot) at the edges of the WM, esp. around the ventricles. WDYT?

Reply:
For the averaging, I imagine the "pre" image would have a lot more blur than the "post"?  

But for the std image: I think that you have me convinced.

Reply:
Okay, let's generate both types of reports - after all, they are pretty cheap to generate. WDYT @slimnsour ?

Reply:
Sounds like a good idea! I'd just need some direction so I can implement it - where is the std map input coming from in the workflow?

Reply:
Just checking up on this @oesteban, you mentioned in the meeting that you would be able to throw something together for the reports you guys were interested in?

Reply:
Yup, sorry I've been booked up. Let's merge this and address the reportlet on a different PR.

Reply:
Sounds good!